lib/mjsonwp/validators.js

```diff
@@ -26,16 +26,20 @@ const validators = {
   },
   timeoutsW3C: (type, ms, scriptDuration, pageLoadDuration, implicitDuration) => {
     // W3C request to send one or more of the following parameters, script/pageLoad/implicit.
-    if (_.isInteger(scriptDuration) || _.isInteger(pageLoadDuration) || _.isInteger(implicitDuration)) {
-      if (_.isInteger(scriptDuration)) {
-        msValidator(scriptDuration);
-      }
-      if (_.isInteger(pageLoadDuration)) {
-        msValidator(pageLoadDuration);
-      }
-      if (_.isInteger(implicitDuration)) {
-        msValidator(implicitDuration);
-      }
+    let hasW3C = false;
+    if (_.isInteger(scriptDuration)) {
```

Comment:
We can use `util.hasValue` verification for the condition, since msValidator is anyway applied there